### PR TITLE
CB-11621 Add windows@4.4.2 release blog post

### PR DESCRIPTION
**Pull request by vladimir-kotikov**

### CODE REVIEW

**Comment by vladimir-kotikov** [comment]
@sgrebnov, @TimBarham, could you please take a look

File: www/_posts/2016-07-27-cordova-windows-4.4.2.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    url: https://github.com/vladimir-kotikov
+title:  "Apache Cordova Windows 4.4.2"
+categories: announcements
+tags: news releases
+---
+
+We are happy to announce that `Cordova Windows 4.4.2` has been released!
+
+This release fixes some issues we've missed in 4.4.1. In particular, we've completely fixed all build issues with
+new VS build tools. For the rest of changes see release notes below.
```
Comment (TimBarham) [inline www/_posts/2016-07-27-cordova-windows-4.4.2.md]: I'm wondering how best to express this. Maybe something like 'In particular, we have fixed build issues experienced with the new install experience in Visual Studio "15" previews'. Do you have any thoughts @mbraude?

File: www/_posts/2016-07-27-cordova-windows-4.4.2.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+Cordova CLI 6.3.0 will automatically start using this version of **Cordova-Windows** when creating new projects.
```
Comment (TimBarham) [inline www/_posts/2016-07-27-cordova-windows-4.4.2.md]: I think this should be "cordova-windows" (no uppercase).

**Comment by TimBarham** [comment]
Couple of comments. Other than that looks good to me.

**Comment by stevengill** [comment]
LGTM

**Pull request closed without merging**